src/Storages/MergeTree/MergeTreePartition.cpp

```diff
@@ -160,11 +160,13 @@ namespace
     };
 }
 
+/*
```

Comment:
Some debugging had been left. Looks like it had been moved into src/Storages/MergeTree/PartMetadataManagerOrdinary.cpp and is redundant here.

Reply:
removed now